src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -301,13 +303,14 @@ export default function VAFacilityPageV2() {
             onSubmit={() =>
               dispatch(routeToNextAppointmentPage(history, pageKey))
             }
-            formContext={{ loadingEligibility, sortMethod }}
+            formContext={{ loadingEligibility, sortMethod, cernerSiteIds }}
```

Comment:
We did this in the old facilities page, but didn't need it in the new one until now